packages/router/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@talend/react-router",
+  "version": "3.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "author": "Talend Frontend <frontend@talend.com> (http://www.talend.com)",
+  "license": "Apache-2.0",
+  "scripts": {
+    "prepublish": "babel -d lib ./src/",
+    "start": "echo nothing to start",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:demo": "jsdoc -c docs.json",
+    "lint:es": "eslint --config ../../.eslintrc --ext .js src",
+    "lint": "npm run lint:es"
+  },
+  "peerDependencies": {
+    "react": "^15.6.2 || ^16.0.0",
+    "react-dom": "^15.6.2 || ^16.0.0"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "8.0.1",
+    "babel-jest": "20.0.3",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-react": "^6.24.1",
+    "enzyme": "^3.1.0",
+    "enzyme-adapter-react-15": "^1.0.1",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "eslint": "^3.6.1",
+    "eslint-config-airbnb": "^11.1.0",
+    "eslint-plugin-import": "^1.16.0",
+    "eslint-plugin-jsx-a11y": "^2.2.2",
+    "eslint-plugin-react": "^6.3.0",
+    "jasmine-immutable-matchers": "1.0.1",
+    "jest": "20.0.3",
+    "jest-cli": "20.0.3",
+    "node-fetch": "^1.7.3",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-test-renderer": "^16.0.0",
+    "redux-mock-store": "^1.2.3",
+    "redux-saga-tester": "^1.0.345",
+    "rimraf": "^2.6.1"
+  },
+  "dependencies": {
+    "@talend/react-cmf": "1.7.0",
```

Comment:
It should be peer dep. Otherwise, we will have to update this dep when we release cmf, or we can end up with 2 react-cmf loaded.
Perhaps it's the same for redux/...

Reply:
ok I have moved it to peer, make sens for cmf because you need cmf to use this one.
I have also pin CMF to >= 1.7